src/org/labkey/test/components/ui/files/FileUploadPanel.java

```diff
@@ -120,18 +125,28 @@ WebElement removeBtn(String fileName)
 
     public static class FileUploadPanelFinder extends WebDriverComponentFinder<FileUploadPanel, FileUploadPanelFinder>
     {
-        private final Locator.XPathLocator _baseLocator = Locator.tagWithClass("div", "file-upload__container").parent();
+        private Locator.XPathLocator _baseLocator = Locator.tagWithClass("div", "file-upload__container").parent();
+        private final boolean _backwardsCompat;
 
         public FileUploadPanelFinder(WebDriver driver)
         {
             super(driver);
+            _backwardsCompat = false;
+        }
+
+        public FileUploadPanelFinder(WebDriver driver, boolean backwardsCompat)
+        {
+            super(driver);
+            _backwardsCompat = backwardsCompat;
+            _baseLocator = Locator.tagWithClass("div", "file-upload--container").parent();
```

Comment:
`backwardsCompat=false` would not work as expected here.
```suggestion
            super(driver);
            _backwardsCompat = backwardsCompat;
            if (backwardsCompat)
                _baseLocator = Locator.tagWithClass("div", "file-upload--container").parent();
```